sandboxed-fs.js

```diff
@@ -4,11 +4,109 @@ const sandboxedFs = {};
 module.exports = sandboxedFs;
 
 const fs = require('fs');
```

Comment:
Imports were placed there by @tshemsedinov, I just decided not to move them anywhere.